spec/system/task_spec.rb

```diff
@@ -1,36 +1,39 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id )}
```

Comment:
associationの定義により、project_idの指定は省略できます。
```suggestion
  let(:task) { create(:task)}
```

Reply:
7f6f618 task全体の定義に変更しました。